registry.json

```diff
@@ -1128,5 +1128,18 @@
     "telegram": "bigdsc",                                            
     "operatorName": "Big",                                
     "github": "BigG-DSC"                                               
-  }
+ },
+ {
+  "ticker": "CROWD",
+  "poolId": "ae96cb925985079c1108a412e810be1871d68cab6dffb6eddbac9adb",
+  "social": {
+    "twitter": "Mendrinos_",
```

Comment:
I cannot find the twitter account.

Reply:
I have updated the commit for your review